packages/jss/benchmark/tests/create-instance.js

```diff
@@ -0,0 +1,18 @@
+import {create} from 'jss'
+import preset from 'jss-preset-default'
+
+// Avoid memory leak with registry.
+const options = {virtual: true}
+
+suite('create a new instance', () => {
+  benchmark('without preset', () => {
+    create(options)
+  })
+
+  benchmark('with preset', () => {
+    create({
+      ...options,
```

Comment:
mb try to create an options before the bench?

Reply:
Moving the options outside the bench only gives you about 2000 ops/sec more.